systems/plants/joints/QuaternionFloatingJoint.cpp

```diff
@@ -48,9 +90,10 @@ void QuaternionFloatingJoint::motionSubspaceDotTimesV(const Eigen::Ref<const Vec
   }
 }
 
-void QuaternionFloatingJoint::randomConfiguration(Eigen::Ref<VectorXd>& q, std::default_random_engine& generator) const
+VectorXd QuaternionFloatingJoint::randomConfiguration(std::default_random_engine& generator) const
 {
-  normal_distribution<double> normal;
+	VectorXd q(7);
```

Comment:
i disagree on this one, because the 7 is also (necessarily) hard-coded below by virtue of setting values q[0] through q[6].